### PR TITLE
show cache status overlays on cache details page (fix #7004)

### LINKED ISSUE #7004: Add an indicator for corrected coordinates in cache details
Hi to all,

Currently you can only tell that a cache has corrected coordinates (geocaching.com), when there is an waypoint "_original location_" present.
It would be really nice to have an indicator at the caches main view, if the cache coordinate is a corrected one.

Edit:

Screenshot for better understanding
screenshot_20180503-211821

### PR DESCRIPTION
**Pull request by moving-bits**
added status overlays on cache details page by using MapUtils.getCacheMarker() function

### CODE REVIEW

**Comment by Lineflyer** [comment]
ok to test

File: main/src/cgeo/geocaching/activity/AbstractActivity.java [hunk 1]
```diff
@@ -241,7 +243,8 @@ private void setCacheTitleBar(@NonNull final CharSequence title, @Nullable final
      * change the titlebar icon and text to show the current geocache
      */
     protected void setCacheTitleBar(@NonNull final Geocache cache) {
-        setCacheTitleBar(TextUtils.coloredCacheText(cache, cache.getName() + " (" + cache.getGeocode() + ")"), cache.getType());
+        setTitle(TextUtils.coloredCacheText(cache, cache.getName() + " (" + cache.getGeocode() + ")"));
```
[the remark this answers is not in the document]
Comment (moving-bits) [reply main/src/cgeo/geocaching/activity/AbstractActivity.java]: setCacheTitleBar called with a String and a cache type does two things:
- display an icon (derived from cache type)
- display the text given

My PR does something similar: setTitle (= display the text) and display an icon, but not the predefined one derived from cache type, but the one given by the MapUtils, which includes the icon overlays. MapUtils implicitly also uses cache.getType

**Comment by Lineflyer** [comment]
After testing the debug build from #7213 I assume we should wait with this change until the permission request changes are merged. The TargetSDK change will also change the layout of the title bar (i.e. the icon is no longer shown at all with the current state of that other PR).

**Comment by moving-bits** [comment]
tested after rebasing it to current master: does NOT work => need to upgrade it first

**Comment by moving-bits** [comment]
adapted to targetSDK26, rebased on today's (Feb 23) master, and squashed

**Comment by Lineflyer** [comment]
@moving-bits The codacy issues are old and fixed, correct?

**Pull request merged**